packages/graphql-yoga/__tests__/error-masking.spec.ts

```diff
@@ -145,12 +145,12 @@ describe('error masking', () => {
     }
   })
 
-  it('non GraphQLError raised in onRequestParse is masked with the correct status code 500', async () => {
+  it('non GraphQLError raised in onPrepare is masked with the correct status code 500', async () => {
```

Comment:
onPrepare looks to generic. onRequestParse is getting triggered after `onRequest`(request received) and when `GraphQLParams` is requested if any other plugin ended the response earlier.

Reply:
`onPrepare` does the "preparation" part, manipulates the params and accepts early results (for caching).

Name seemed like a right fit IMHO; but debatable, I agree.